File: codingbat/python/src/list-2/sum67.py
def sum67(nums):
  sum = 0;
  no_skip = True;
  i = 0;
  while i < len(nums):
    if no_skip and nums[i] != 6:
      sum += nums[i];
    if nums[i] == 6:
      no_skip = False;
    if nums[i] == 7:
      no_skip = True;
    i += 1;
  return sum;

File: codingbat/java/src/array-2/sum67.java
public int sum67(int[] nums) {
  boolean count = true;
  int c = 0;
  for(int i=0; i<nums.length; i++){
    if(nums[i] == 6){
      count = false;
    }
    c += count ? nums[i] : 0;
    if(nums[i] == 7){
      count = true;
    }
  }
  return c;
}
